css/signin.css

```diff
@@ -0,0 +1,121 @@
+*{
+    box-sizing: content-box;
+}
+/* 심화 : color 변수로 지정하기 */
+:root{
+    --linkbrary-bg : #f0f6ff;
+    --linkbrary-primary-color :#6D6AFE;
+    --gra-purpleblue-to-skyblue:linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%);
+    --grey-light: #F5F5F5;
+    --linkbrary-gray-20: #CCD5E3;
+    --linkbrary-gray-10: #E7EFFB;
+    --linkbrary-gray-100: #373740;
+}
+html{
+    height: 100%;
+    font-size: 62.5%;
+    /*10px*/
+}
+
+body{
+    display: flex;
+    background-color:var(--linkbrary-bg) ;
+    min-height: 100%;
+    flex-direction: column;
+    margin-top: 10%;
+}
+
+.wrap{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #F0F6FF;
+    gap: 3rem;
+    font-size: 1.6rem;
+
+}
+
+header{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1.6rem;
+}
+
+.header__logo-img{
+    width: 21rem;
+    height: 4rem;;
+}
+header p{
+    margin: 0;
+    font-weight: 400;
+}
+.header__signup-link{
+    margin-left: 0.8rem;
+    font-weight: 600;
+}
+
+.form__signin{
+    display: flex;
+    width: 40rem;
+    flex-direction: column;
+    justify-content: center;
+    align-items: left;
+}
+.form__signin label{
+    font-size: 1.4rem;
+}
+.form__signin input{
+    margin: 1.2rem 0 2.4rem;
+    padding: 1.8rem 1.5rem;
+    border: 0.1rem solid var(--linkbrary-gray-20);
+    border-radius: 0.8rem;
+}
+.form__signin input:focus{
+    border: var(--linkbrary-primary-color);
```

Comment:
.form__signin input:focus-visible{
    outline-color: var(--linkbrary-primary-color);
}

1. focus vs focus-visible
https://codepen.io/michai/pen/eYgLPBG
https://pawelgrzybek.com/the-difference-between-css-focus-and-focus-visible-pseudo-class/